src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -208,6 +220,49 @@ private BidRequest createSingleRequest(Imp imp, BidRequest bidRequest, boolean u
                 .build();
     }
 
+    private String makeUri(BidRequest bidRequest) {
+        final String tkXint = tkXintValue(bidRequest);
+        if (!StringUtils.isBlank(tkXint)) {
+            try {
+                final URIBuilder uriBuilder = new URIBuilder(endpointUrl);
+                final List<NameValuePair> queryParams = uriBuilder.getQueryParams();
+                return uriBuilder.clearParameters()
+                        .setParameters(updatedQueryParameters(queryParams, tkXint))
+                        .build().toString();
+            } catch (URISyntaxException e) {
+                throw new PreBidException("Cant resolve endpoint parameters: " + endpointUrl, e);
```

Comment:
Throwing exception is too harsh in this case. Let's log warning and just proceed with unchanged endpoint URL.